ui/components/ui/add-network/add-network.component.js

```diff
@@ -0,0 +1,116 @@
+import React, { useContext } from 'react';
```

Comment:
Please drop the .component suffix from all files of new components. This is an old pattern we shouldn't extend.

Reply:
Fixed